api/v1/installation_types.go

```diff
@@ -198,8 +198,30 @@ type InstallationSpec struct {
 	// Kubernetes Service CIDRs. Specifying this is required when using Calico for Windows.
 	// +optional
 	ServiceCIDRs []string `json:"serviceCIDRs,omitempty"`
+
+	// Azure is used to configure azure provider specific options.
+	// +optional
+	Azure *Azure `json:"azure,omitempty"`
+}
+
+type Azure struct {
+	// PolicyMode determines whether the "control-plane" label is applied to namespaces. It offers two options: Default and Manual.
+	// The Default option adds the "control-plane" label to namespaces that have the PodSecurityStandard set to privileged.
```

Comment:
Why are we making this dependent on the PodSecurityStandard too? Did I miss something in the design ore previous discussions that PodSecurityStandard was going to be a condition on setting the label?

Reply:
Yeah, these shouldn't be directly tied to each other. We just brought that up as another similar configuration.

Reply:
Are you suggesting to add label to all the namespaces in Default option ?
In the meeting we decided to add it only to the privileged namespaces

Reply:
Nope, not all namespaces. 

We should only add it to namespaces that need this policy configured - that may or may not be the same set of namespaces as those that require PSSPrivileged, but the point is that doesn't mean hard equality between `control-plane` and `PSSPrivileged`, and it definitely isn't something the user should know about / be able to rely on.

Reply:
Got it, I'll update the documentation.